### PR TITLE
Updates to token stakedrop documentation

### PR DESCRIPTION
**Pull request by pdyraga**
Some updates to the documentation so that the information on how
KEEP token balances are resolved is more precise.

### CODE REVIEW

File: token-stakedrop/README.adoc [hunk 1]
```diff
@@ -1,70 +1,87 @@
-# KEEP Token Tracker
+# Overview
 
-This is a tool for tracking KEEP Token ownership. It will walk over predefined
-sources of truth and inspect KEEP Token ownership at a specific block number.
+This is a tool for finding KEEP token owners on Ethereum (or on any other 
+EVM-compatible chain). It will walk over predefined sources of truth
```
Comment (dimpar) [inline token-stakedrop/README.adoc:4]: It iterates over sources of truth (predefined contracts and individual accounts) to find KEEP token owners at a target block number.

File: token-stakedrop/README.adoc [hunk 2]
```diff
@@ -1,70 +1,87 @@
...
+(information) and lookup KEEP token owners at a specific block number.
+The tool generates a Merkle tree used as an input to
+https://github.com/keep-network/keep-core/blob/master/solidity/contracts/token-distribution/TokenDistributor.sol[`TokenDistributor`]
+contract airdropping another ERC20 token with amounts based on KEEP token
```
Comment (dimpar) [inline token-stakedrop/README.adoc:8]: ... with amounts equal to KEEP token balances.

File: token-stakedrop/README.adoc [hunk 3]
```diff
@@ -1,70 +1,87 @@
...
+balances. 
 
-The tool returns a map of addresses with their total KEEP Token holdings.
-
-## Sources Of Truth
+# Sources of truth
 
 Holders of the tokens can be externally owned accounts or contracts. It is
-necessary to resolve actual owner for tokens held by a contract.
+necessary to resolve actual owner of tokens held by a contract.
```
Comment (dimpar) [inline token-stakedrop/README.adoc:14]: an actual owner

File: token-stakedrop/README.adoc [hunk 4]
```diff
@@ -1,70 +1,87 @@
...
+https://github.com/keep-network/keep-core/blob/master/solidity/contracts/token-distribution/TokenDistributor.sol[`TokenDistributor`]
+contract airdropping another ERC20 token with amounts based on KEEP token 
+balances. 
 
-The tool returns a map of addresses with their total KEEP Token holdings.
-
-## Sources Of Truth
+# Sources of truth
 
 Holders of the tokens can be externally owned accounts or contracts. It is
```
Comment (dimpar) [inline token-stakedrop/README.adoc:13]: Holders of KEEP tokens...

**Review by nkuba — APPROVED** [review]
LGTM, I left suggestions.

**Review by dimpar — COMMENTED** [review]
Minor comments, tomayto-tomahto type of thing. Up to you if you wanna change it :)

File: token-stakedrop/README.adoc (hunk 1 above)
Comment (pdyraga) [reply token-stakedrop/README.adoc:4]: In this case, source of truth is one of the components looking up the information. We have a separate source of truth for KEEP liquid balances, separate source for token grants and so on. This is how @nkuba's code structured it.

File: token-stakedrop/README.adoc (hunk 2 above)
Comment (pdyraga) [reply token-stakedrop/README.adoc:8]: Not sure if "equals" work here - some sources, like LPs staked are more complicated. WDYT?

File: token-stakedrop/README.adoc (hunk 4 above)
Comment (pdyraga) [reply token-stakedrop/README.adoc]: https://github.com/keep-network/keep-core/pull/2400/commits/4452560d071fd3aa60b3966c7ea79afe12ada6d4

File: token-stakedrop/README.adoc (hunk 3 above)
Comment (pdyraga) [reply token-stakedrop/README.adoc]: https://github.com/keep-network/keep-core/pull/2400/commits/4452560d071fd3aa60b3966c7ea79afe12ada6d4

File: token-stakedrop/README.adoc [hunk 5]
```diff
@@ -1,70 +1,87 @@
...
+for any accounts that have ever held KEEP tokens. Iterates over the set of
+addresses and checks KEEP token balance at the specific target block.
 
-To avoid duplication with results of other source of truths lookups it ignores
-addresses that are contracts handled by the other sources. For the ignored contracts
-separate sources of truth should be implemented.
+To avoid duplication with results of lookups of other sources of truth, it ignores
+addresses of contracts handled by them. For the ignored contract addresses, separate
+sources of truth are implemented. For example, Token Grant and Token Staking
+contract addresses are ignored as the actual owners of KEEP tokens
```
[the remark this answers is not in the document]
Comment (pdyraga) [reply token-stakedrop/README.adoc]: https://github.com/keep-network/keep-core/pull/2400/commits/4452560d071fd3aa60b3966c7ea79afe12ada6d4

File: token-stakedrop/README.adoc [hunk 6]
```diff
@@ -1,70 +1,87 @@
...
-target block.
+Scans `KeepToken` contract (`0x85Eee30c52B0b379b046Fb0F85F4f3Dc3009aFEC`) 
+for any accounts that have ever held KEEP tokens. Iterates over the set of
+addresses and checks KEEP token balance at the specific target block.
 
-To avoid duplication with results of other source of truths lookups it ignores
-addresses that are contracts handled by the other sources. For the ignored contracts
-separate sources of truth should be implemented.
+To avoid duplication with results of lookups of other sources of truth, it ignores
+addresses of contracts handled by them. For the ignored contract addresses, separate
```
[the remark this answers is not in the document]
Comment (pdyraga) [reply token-stakedrop/README.adoc]: https://github.com/keep-network/keep-core/pull/2400/commits/4452560d071fd3aa60b3966c7ea79afe12ada6d4

File: token-stakedrop/README.adoc (hunk 1 above)
Comment (pdyraga) [reply token-stakedrop/README.adoc:4]: https://github.com/keep-network/keep-core/tree/master/token-stakedrop/src/truth-sources

Comment (dimpar) [reply token-stakedrop/README.adoc:4]: Yea, I know that, that's why I'm saying "it iterates over sources of truth.. "

Comment (pdyraga) [reply token-stakedrop/README.adoc:4]: Ah, I missed the suggestion in the diff. Will change.

Comment (pdyraga) [reply token-stakedrop/README.adoc]: Ended up with https://github.com/keep-network/keep-core/pull/2400/files#diff-a0b8a21602c359b9faff24ed6a5b2514bd7e4360b7c483756e817034b961107a

File: token-stakedrop/README.adoc (hunk 2 above)
Comment (dimpar) [reply token-stakedrop/README.adoc:8]: Okay, I see. I think when we say "based" in this context is generic and whoever reads it, doesn't know how much they will get, might be 1 but maybe other number. When we say "equal", then it's 1:1. In the overview we just say that whatever KEEP we found (doesn't matter how) then we will sort match it.
In the LP Rewards description we go into the more details about how we find KEEP tokens, and the finding is "based" on the LP tokens.
Idk.. if you think it's okay to use "based" I'm okay with it too :)

**Review by nkuba — APPROVED** [review]

File: token-stakedrop/README.adoc (hunk 2 above)
Comment (pdyraga) [reply token-stakedrop/README.adoc:8]: Yeah, I'd keep "based" for the exact reason you mentioned, I'd keep the precise information under the particular source of truth and be more generic earlier.

**Review by dimpar — APPROVED** [review]

**Pull request merged**